lib/msal_auth.dart

```diff
@@ -106,4 +106,8 @@ class MsalAuth {
       throw e.msalException;
     }
   }
+
+  Future<void> setLoginHint(String loginHint) async {
```

Comment:
I don't think we require an external function, use the argument in the existing config and setup accordingly.

Reply:
It should be like..

```dart
await MsalAuth.createPublicClientApplication(
  loginHint: "my value" // it will be optional field.
  ....
)
```